docs/concepts/configuration/manage-compute-resources-container.md

```diff
@@ -370,11 +370,62 @@ For container-level isolation, if a Container's writable layer and logs usage ex
 
 ## Extended Resources
 
-Kubernetes version 1.8 introduces Extended Resources. Extended Resources are
-fully-qualified resource names outside the `kubernetes.io` domain. Extended
-Resources allow cluster operators to advertise new node-level resources that
-would be otherwise unknown to the system. Extended Resource quantities must be
-integers and cannot be overcommitted.
+Extended Resources are fully-qualified resource names outside the
+`kubernetes.io` domain. They allow cluster operators to advertise and users to
+consume the non-Kubernetes-built-in resources, such as GPUs and software
+licenses.
+
+There are two steps required to use Extended Resources. First, the cluster
+operator must advertise an Extended Resource. Second, users must request the
+Extended Resource in Pods.
+
+### Managing extended resources
+
+#### Node-level extended resources
+
+Node-level extended resources (e.g., GPUs) are tied to nodes. They are usually
+managed by the [Device
+Plugin](https://kubernetes.io/docs/concepts/cluster-administration/device-plugins/)
+running on each node.
+
+#### Cluster-level extended resources
+
+Cluster-level extended resources (e.g., software licenses) are not tied to
+nodes. They are usually managed by scheduler extenders, which handle the
+resource comsumption, quota and so on.
+
+You can specify the extended resources that are handled by scheduler extenders
+in [scheduler policy
+configuration](https://github.com/kubernetes/kubernetes/blob/release-1.10/pkg/scheduler/api/v1/types.go#L31).
+
+**Example:**
+
+The below scheduler policy configuration indicates that the cluster-level
+extended resource "example.com/foo" is handled by scheduler extender.
+ - The scheduler will only send a pod to the scheduler extender iff the pod
+   requests "example.com/foo".
+ - The `ignoredByScheduler` indicates that scheduler will skip checking the
```

Comment:
s/indicates that scheduler will skip checking/field specifies that the scheduler does not check

Reply:
Done.